### DISCUSSION TITLE
关于32位WIN2008服务器版启动器安装的问题

### DISCUSSION THREAD

**Discussion by lxiang2000** [natfrp/wiki#485] [使用问题]
我有一台32位的win2008服务器版操作系统的主机，想安装启动器，但是好像不支持该系统，请问有什么办法能安装吗？或者那个frpc能代替吗？

**Comment by fakeboboliu** [comment]
请使用 ws2008r2 或 ws2016 或更高，对于目前不在支持内的系统我们无法给您一个确切的答复。
